style/haskell/README.md

```diff
@@ -5,13 +5,16 @@ Haskell
 
 * Break long expressions before operators or keywords.
 * Break long type signatures between arguments.
+* Don't use a blank line between pragma and module statements.
```

Comment:
I don't have much of an opinion on it, but why no blank line? I usually add one when I write Haskell.